tests/utils/network_setup.go

```diff
@@ -213,13 +219,11 @@ func DeployTeleporterContracts(transactionBytes []byte, deployerAddress common.A
 	ctx := context.Background()
 
 	for _, subnetInfo := range subnetsInfo {
-		client := subnetInfo.ChainWSClient
-
 		// Fund the deployer address
 		{
 			fundAmount := big.NewInt(0).Mul(big.NewInt(1e18), big.NewInt(10)) // 10eth
 			fundDeployerTx := createNativeTransferTransaction(ctx, subnetInfo, fundedAddress, fundedKey, deployerAddress, fundAmount)
-			SendTransactionAndWaitForAcceptance(ctx, subnetInfo.ChainWSClient, fundDeployerTx)
+			SendTransactionAndWaitForAcceptance(ctx, subnetInfo.ChainWSClient, subnetInfo.ChainRPCClient, fundDeployerTx)
```

Comment:
With these changes, I think of a few of the methods under `utils` here have become specific to running the E2E tests against a local network. Is there a way we could make that more clear?

Reply:
All of the methods here are specific to the local network setup. I've renamed this file `local_network_setup.go` to make that clear. I've also moved some of the functions only used by the local network test from `utils.go` to a new file `local_network_utils.go`

Reply:
thoughts on moving the `local_network_setup` and `local_network_utils` files under `tests/network`? Could possibly even make it `tests/network/local/` and `tests/network/fuji/`

Reply:
bumping above ^

Reply:
Elected to organize the files according to this comment: https://github.com/ava-labs/teleporter/pull/93#discussion_r1393131946